src/pages/jaewonyang/Main/Comment/CommentJaeWon.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './CommentJaeWon.scss';
+
+function CommentJaeWon({ commentList, key }) {
+  return (
+    <>
+      <ul className="CommentJaeWon">
+        {commentList.map(e => (
```

Comment:
매개변수명도 어떤 데이터인지 알 수 있게 짓기
```suggestion
        {commentList.map(comment => (
```